docs/modules/deploy/pages/configuring-kubernetes.adoc

```diff
@@ -94,6 +94,7 @@ There are several properties to configure, all of which are optional.
 
 * `namespace`: Kubernetes Namespace where Hazelcast is running; if not specified, the value is taken from the environment variables `KUBERNETES_NAMESPACE` or `OPENSHIFT_BUILD_NAMESPACE`. If those are not set, the namespace of the Pod will be used (retrieved from `/var/run/secrets/kubernetes.io/serviceaccount/namespace`).
 * `service-name`: service name used to scan only Pods connected to the given service; if not specified, then all Pods in the namespace are checked
+  NOTE: If you don't specify `service-name` and fall back to a `namespace` only discovery, all pods in the namespace must be Hazelcast member pods; any other kind of Pod will block member discovery process of the Hazelcast member pods. 
```

Comment:
I think we do not have that requirement exactly. We should warn the users that existence of other pods might prevent the cluster from being formed. MC prevented that because it could probably talk to Hazelcast more than any other random pod. But I doubt it would happen with any other pod.

Reply:
... also MC port blocks things even without cluster configs (so while just listening on 8080)

Reply:
Ah you might be right. But I would still be hesitant to put `will block` phrase into the documentation since we are only saying this by experience and not by theory, `might block` may be better IMO.